channels/us.m3u

```diff
@@ -1854,3 +1854,5 @@ https://thegateway.app/YouToo/CueTones/playlist.m3u8
 https://thegateway.app/YouToo/YTamerica/playlist.m3u8
 #EXTINF:-1 tvg-id="KanalDisney.us" tvg-country="RU" tvg-language="Russian" tvg-logo="https://i.imgur.com/Q9KoVy9.png" group-title="Kids",Канал Disney (576p) [Not 24/7]
 http://188.40.68.167/russia/disney/playlist.m3u8
+#EXTINF:-1 tvg-id="ABCNewsAustralia.us" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/bLzWAtX.jpg" group-title="News",ABC News (Australia)
+https://ask-streamlink.herokuapp.com/iptv-query?streaming-ip=https://www.youtube.com/watch?v=W1ilCy6XrmI/live
```

Comment:
![image](https://user-images.githubusercontent.com/7253922/136207464-3fe3775b-9bb1-445e-ac91-91bd3e3744fe.png)

Reply:
429 Too Many Requests. It seems that someone tried too hard.

Reply:
Yeah, I'm getting the same error. Is that coming from Youtube, or Heruko?

Reply:
Seems the links I tested going to YouTube from my Heroku app IP, will not work. 
When I use another URL's it does. Can someone confirm with a link other than YouTube?